components/docker-auth/pkg/authserver/authz_handler.go

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package authserver
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/cellery-io/cellery-hub/components/docker-auth/pkg/extension"
+)
+
+func dbConn() (*sql.DB, error) {
+	dbDriver := extension.MYSQL_DRIVER
+	dbUser := os.Getenv(extension.MYSQL_USER_ENV_VAR)
+	dbPass := os.Getenv(extension.MYSQL_PASSWORD_ENV_VAR)
+	dbName := extension.DB_NAME
+	host := os.Getenv(extension.MYSQL_HOST_ENV_VAR)
+	port := os.Getenv(extension.MYSQL_PORT_ENV_VAR)
+
+	db, err := sql.Open(dbDriver, fmt.Sprint(dbUser, ":", dbPass, "@tcp(", host, ":", port, ")/"+dbName))
+	if err != nil {
+		log.Println("Error occurred while connecting to the database")
```

Comment:
When logging it's better to log the error as well. (Please fix in all places)

When returning the error, it's best to wrap the error with a bit more information. Otherwise it might be hard to pinpoint exactly what happened. (Please fix in all places)